code/style.css

```diff
@@ -1,7 +1,574 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+/* MOBILE VIEW EXTRA SMALL DEVICES max-width: 360px */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Raleway', sans-serif;
+}
+
+/* HEADER/HERO IMAGE NAVBAR CONTAINER WITH LOGO, HAMBURGER & NAV MENU*/
+/* 
+.container {
+  width: 100%;
+  align-self: flex-end;
+} */
+
+.navbar-container {
+  padding: 20px 20px;
+  margin: auto;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between; 
+}
+
+.navbar-container nav {
+  display: none;
+} 
+
+.title {
+  padding: 20px;
+  margin: 5px;
+  width: 100px;
+  background: rgb(230, 243, 229);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.title h1 {
+  font-family: 'Raleway', sans-serif;
+  padding: 0px;
+  margin: 0px;
+  line-height: 1.2;
+  color: #0e2708;
+  font-size: 20px;
+}
+
+.hamburger-menu {
+  width: 40px;
+  height: 30px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  margin-right: 20px;
+}
+
+.hamburger-menu div {
+  height: 4px;
+  width: 40px;
+  border-radius: 5px;
+  background: #31482c;
+}
+
+/* HERO IMAGE WITH TEXT & BUTTON */
+
+.hero-image {
+  max-width: 100%;
+  height: 550px;
+  background-image: url("Hero-image-1.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0; 
+  z-index: -1;
+}
+
+.hero-image-details {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 70px;
+  padding:10px;
+  
+}
+
+.hero-image-text {
+  padding: 10px;
+  margin-bottom: 10px;
+  opacity: 0.9;
+  display: flex;
+  position: relative;
+  background: #ffffff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.hero-image-text h1 {
+  color: #0e2708 ;
+  font-family: 'Rozha One', serif;
+  font-size: 50px;
+  text-align: left;
+  margin: 5px;
+  padding: 10px 0px;
+  font-weight: 400;
+  border-bottom: solid #90be6b 2px;
+  line-height: 01em;
+  text-align: center;
+}
+
+.hero-image-text h3 {
+  font-family: 'Raleway', sans-serif;
+  color:#0e2708;
+  font-size: 20px;
+  font-weight: 200;
+}
+
+header button {
+  height: 50px;
+  border: none;
+  background: #f0c697;
+  color: #ffffff;
+  padding: 10px;
+  margin: auto;
+  position: relative;
+  box-sizing: border-box;
+}
+
+header button:hover {
+  cursor: pointer;
+  height: 60px;
+  width: 90px;
+  transition: 2s;
```

Comment:
This seems like quite a long time?

Reply:
Ok I see :) So really like 1s is enough? I need to learn more about these transition times I think. Something for further reading.